### PR TITLE
cachetools

### PR DESCRIPTION
**Pull request by maartenbreddels**

### CODE REVIEW

File: recipes/cachetools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  run:
+    - python
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  imports:
+    - cachetools
+
+about:
+  home: http://github.com/simplejson/simplejson
```
Comment (dopplershift) [inline recipes/cachetools/meta.yaml]: Probably need to update this line.

Comment (maartenbreddels) [inline recipes/cachetools/meta.yaml]: 👍  thanks

File: recipes/cachetools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/cachetools/meta.yaml]: Please remove all the comments from the example.

File: recipes/cachetools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{% set name = "cachetools" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "715a7202240dc20dbe83abdb2d804d543e2d4f07af146f53c82166bd75f3a628" %}
+# run to calculate checksum `openssl sha256 <file name>`.
```
Comment (jakirkham) [inline recipes/cachetools/meta.yaml]: Please drop this comment.

File: recipes/cachetools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - cachetools
+
+about:
+  home: https://github.com/tkem/cachetools
+  license: MIT
```
Comment (jakirkham) [inline recipes/cachetools/meta.yaml]: Could add `license_family: MIT`.

Comment (jakirkham) [inline recipes/cachetools/meta.yaml]: Done.

File: recipes/cachetools/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  summary: 'Extensible memoizing collections and decorators'
+
+  description: |
+    This module provides various memoizing collections and decorators, including variants of the Python 3 Standard Library @lru_cache function decorator.
+  doc_url: http://pythonhosted.org/cachetools/
+  dev_url: https://github.com/tkem/cachetools
+
+extra:
+  recipe-maintainers:
+    - MaartenBreddels
```
Comment (jakirkham) [inline recipes/cachetools/meta.yaml]: Please add terminal newline.

Comment (jakirkham) [inline recipes/cachetools/meta.yaml]: Done.

**Comment by maartenbreddels** [comment]
Can this be merged?

**Comment by 183amir** [comment]
Thanks @maartenbreddels If this is your first time adding a recipe here, I guess @jakirkham will comment and explain things for you.

**Comment by maartenbreddels** [comment]
It's the 3rd now I think in a few days, need all this for #1600, and it helps others as well, maybe also #1724 needs a bump, I guess it's an easy one as well 😄

**Pull request merged**